docs/dev/Creating UI5 Web Components Packages.md

```diff
@@ -107,42 +107,42 @@ That's it!
 
 ## Understanding the project structure
 
-### package.json
+### Package.json
```

Comment:
package.json is the name of the file, should not be renamed

Reply:
It's also a title but if you think this may cause more confusion, then we'll not capitalize it.

Reply:
Then, `The package.json` perhaps?, or something like this, yes I would keep `package.json` as it is.

Reply:
If it's tagged as a technical name, it's OK to be written in lowercase and it wouldn't need the article.